addons/website_event/views/event_registration_views.xml

```diff
@@ -64,4 +75,16 @@
             </field>
         </field>
     </record>
+
+    <record id="event_registration_view_attendee_search_inherit_online" model="ir.ui.view">
+        <field name="name">event.registration.view.attendee.search.inherit.online</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.event_registration_view_attendee_search"/>
+        <field name="arch" type="xml">
+            <field name="registration_answer_ids" position="replace"/>
```

Comment:
Well seems we are adding an inherited view just for an ordering or a string ? In which case would say: tant pis, isn't it ?

Reply:
I created the `event_registration_view_attendee_search` mainly because jija ask for a search view different from the one of the other attendee actions.
`=> It should not be the same view as other attendee actions (reporting, ...) ` 
However if I can just modify the `view_registration_search` it's indeed easier

Reply:
And event_registration_view_search_event_specific ? As it is already stripping some event related fields ...

Reply:
You mean I could make the `event_registration_view_attendee_search` inherit from `event_registration_view_search_event_specific` to have less fields to remove ? 

But the `event_registration_view_attendee_search` needs the event things like the `event_id` , the `filter_is_ongoing` filter and the `group_event` which are removed by the `event_registration_view_search_event_specific` ... 

Basically the `event_registration_view_attendee_search` is the same as the `view_registration_search` but with a modified domain, a reorder, without the `event_user_id` and the `event_organizer_id` and with the `event_ticket_id` field. That's why I inherited from this one.

Reply:
I don't understand, 'event_registration_view_attendee_search' removes event_id and you say you need it ?

Anyway I would say "participant" and "ticket" could be standard. Seems we could simplify a bit all those, having too much different views is misleading and does not help maintaining the module. It seems to me we need two kind of views: one generic (with event filters) and one event-specific (hiding event fields), that you will use in the context of a given event (would be 'event_registration_view_search_event_specific'). I still don't see much differences between your new view and event_registration_view_search_event_specific (could probably merge both).

Reply:
The attendee view needs the event search things. I removed the `event_id` but I declared it again a few lines underneath to get the order I want...
```xml
    <record id="event_registration_view_attendee_search" model="ir.ui.view">
        <field name="name">event.registration.view.attendee.search</field>
        <field name="model">event.registration</field>
        <field name="inherit_id" ref="view_registration_search"/>
        <field name="mode">primary</field>
        <field name="arch" type="xml">
            <xpath expr="//search/field[@name='event_id']" position="replace"/> <===== REMOVED HERE
            <xpath expr="//search/field[@name='event_user_id']" position="replace"/>
            <xpath expr="//search/field[@name='event_organizer_id']" position="replace"/>
            <xpath expr="//search/field[@name='name']" position="replace"/>
            <xpath expr="//search/field[@name='partner_id']" position="replace"/>
            <xpath expr="//search/field[@name='company_id']" position="replace"/>
            <xpath expr="//search/field[@name='id']" position="after">
                <field name="name" string="Participant"
                    filter_domain="['|', '|', ('name', 'ilike', self), ('email', 'ilike', self), ('company_name', 'ilike', self)]"/>
                <field name="company_id"/>
                <field name="partner_id"/>
                <field name="event_ticket_id" string="Ticket"/>
                <field name="event_id"/>  <========= ADDED HERE
            </xpath>
        </field>
    </record>
```

The attendee view is not event specific, we get all attendee registrations of every events. You still want to filter on the event_id in this view to only get the attendee registrations of a specific event so we need the event filters.

The event specific view on the other hand is the one opened from the event form so it is specific to a certain event and we don't need all the filtering on the event stuff.

Do you get what I mean ? :sweat_smile:  sorry if it isn't clear enough

Reply:
I will make the modifications on the `view_registration_search` view and remove my `event_registration_view_attendee_search` then. It will maybe be easier

Reply:
Arf, did not see it was added back... playing with specific views is quickly complete mess, especially with inheritance.

Then if you search on event, why removing fields like "event_organizer_id" ? Or we consider it is useless and we remove it from standard view, anyway you can find them in advanced filters.

If you merge with 'view_registration_search' it will be easier yes. A new view just to reorder things is not necessary (and in the end I think it is easier to have only 1 search view otherwise you end up wondering why sometimes it is proposed, somerimes not).

Reply:
I removed the `event_organizer_id` and the `event_user_id`  based on this spec: 
```
<field name="event_user_id" string="Responsible"/>
===> Apparently this is linked to the next activity? Does not seem EVER needed, let's remove this field in master 

<field name="event_organizer_id" string="Organizer"/>
===> This one is OK but not needed in the search, you will not search for Pauline when registering people at the OXP :) 
```

Ok i'll merged with the `view_registration_search` :) 
Thx for your time

Reply:
Yep, it we don't need them they can be found in advanced filters anyway :)